boto3/s3/inject.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from boto3.s3.transfer import S3Transfer
+
+
+def inject_s3_transfer_methods(class_attributes, **kwargs):
+    class_attributes['upload_file'] = upload_file
+    class_attributes['download_file'] = download_file
```

Comment:
Should we check for name collisions here? Seems unlikely to happen, but getting an error from the integration tests might be nice if it ever does.